src/rsa.rs

```diff
@@ -86,9 +92,29 @@ impl signature_impl::VerificationAlgorithmImpl for RSA_PKCS1 {
     }
 }
 
+macro_rules! rsa_pkcs1_padding {
+    ( $PADDING_ALGORITHM:ident, $digest_alg_name:expr,
+      $digest_alg:expr, $digestinfo_prefix:expr ) => {
+
+        #[doc="PKCS#1 1.5 padding with the "]
+        #[doc=$digest_alg_name]
+        #[doc=" digest algorithm."]
```

Comment:
We need to have a simlar "not available in "no_heap" mode comment.

Also, it seems better to just fold this macro into rsa_pkcs1. I doubt that we'll ever have a mismatch between the padding types we support for signing and the types we support for verification.